src/components/WwwFrame/SearchBar.vue

```diff
@@ -240,6 +240,7 @@ form.search-form {
 	}
 
 	.search-icon {
+		fill: $kiva-text-dark;
```

Comment:
I think we should move the base styles for icon sets (some stroke, some fills) out of KvIcon to a global inclusion so these individual tweaks wouldn't be required unless you want a special color. It's not very much css.

Reply:
That's not necessary. The css will only be included once per page, as css is included per component definition, not per component instance. Also, as this is in `src/components` it's probably already being bundled into the global css.